src/applications/vaos/components/calendar/CalendarRadioOption.jsx

```diff
@@ -1,29 +1,30 @@
 import React from 'react';
+import moment from 'moment';
 
-const CalendarRadioOption = ({
-  id,
-  fieldName,
-  value,
-  checked,
-  onChange,
-  label,
-}) => (
-  <div className="vaos-calendar__option">
-    <input
-      id={`radio-${id}`}
-      type="radio"
-      name={fieldName}
-      value={value}
-      checked={checked}
-      onChange={onChange}
-    />
-    <label
-      className="vads-u-margin--0 vads-u-font-weight--bold vads-u-color--primary"
-      htmlFor={`radio-${id}`}
-    >
-      {label}
-    </label>
-  </div>
-);
+const CalendarRadioOption = ({ id, fieldName, value, checked, onChange }) => {
+  const time = moment(value);
+  const meridiem = time.format('A');
+  const screenReaderMeridiem = meridiem.replace(/\./g, '').toUpperCase();
+
+  return (
+    <div className="vaos-calendar__option">
+      <input
+        id={`radio-${id}`}
+        type="radio"
+        name={fieldName}
+        value={value}
+        checked={checked}
+        onChange={onChange}
+      />
+      <label
+        className="vads-u-margin--0 vads-u-font-weight--bold vads-u-color--primary"
+        htmlFor={`radio-${id}`}
+      >
+        {time.format('h:mm')} <span aria-hidden="true">{meridiem}</span>{' '}
```

Comment:
Interested in thoughts about putting this here, since it sort of breaks the idea that these radio and checkbox components are just basic form components and adds more specific calendar functionality to them.

Reply:
This is a tough call.  I started this Calendar widget with the hope/idea it could be reusable.  But as requirements got added, it became more and more difficult to keep it agnostic.  We could just leave it as it is now and make new task to make it more reusable in the future.

The other option is we could adjust the `getOptionsByDate()` in the `additionalOptions` CalendarWidget prop to return a content block instead of just a label string (e.g. line 19 `src/applications/vaos/components/DateTimeSelectField.jsx`).  Would require a bit of refactoring.

Reply:
Yeah, I think you could make that work. That also feels weird to me, though, since it's not obvious why you'd have markup generated there rather than in the component where it's being used.

It's tough to make something like this generic/reusable when we don't have a clear idea of the scope of who would use it and what features they would and wouldn't want. I agree with the idea of working through some of that in a separate ticket. We can come up with the parts of it we think should be generic and then refactor from there.

Reply:
Yeah good point.  Another option could be to add a `hiddenText` field on the options, so we could avoid having markup there. But yeah, handling all the different parts to make it generic in a separate tickets sounds good to me.

Reply:
I created a ticket for that work, and also changed the way I was doing this to follow the pattern you had set up previously, since we can make better decisions about what to refactor or not after we think through what the widget should do.